buddyboss/buddyboss-platform
# Fix - Activity comment more option icon overlaps dropdown

### All Submissions:


Fixes #2726 

### How to test the changes in this Pull Request:

1. Go to activity
2. Check any activity with multiple comments
3. Click on comment more option dropdown

### Proof Screenshots or Video
https://ibb.co/285HbkT

### Other information:
